### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VaibhavSinghDev**
Fixed some typos.

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Currently planning to use:
 
 However, this is by no means set in stone!
 
-# how to contribute
+# How to contribute ?
```
Comment (ykdojo) [inline README.md:18]: no space before the question mark?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ykdojo) [reply README.md:18]: actually let's do no question mark for this one

**Comment by ykdojo** [comment]
I realized this is a duplicate of this one: https://github.com/ykdojo/defaang/pull/22/files

**Comment by VaibhavSinghDev** [comment]
> I realized this is a duplicate of this one: https://github.com/ykdojo/defaang/pull/22/files

Oh  okay np!

**Pull request closed without merging**